### PR TITLE
extract dataset specific configuration to own files

### PR DESCRIPTION
**Pull request by sgratzl**
based on #74 

generalizes the idea of dataset definitions by extraction almost all dataset specific code into one place. 

extras: 
 * use the selected objects rather than strings in the select boxes. e.g. the dataset object instead of the key only
 * use React to render the tooltip -> allowing fat tooltips
 * store the selected node object instead of the name
 * use different node scale scales (linear, power, log, sqrt) instead of hard coded factors, might require fine tuning regarding the scale range

### CODE REVIEW

File: src/components/graph-vis/info-panel.js [hunk 1]
```diff
@@ -21,25 +20,26 @@ const visStyles = theme => ({
   }
 });
 
+
+
```
Comment (waxlamp) [inline src/components/graph-vis/info-panel.js]: Strike the extra blank line.

File: src/datasets/utils.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            }
+        };
+    }
+}
+
+/**
+ * coloring factory by property existence
+ * @param {string} prop property to visualize
+ */
+export function booleanColorFactory(prop, existent, notExistent) {
```
Comment (waxlamp) [inline src/datasets/utils.js]: Should the parameters be called `discovered` and `undiscovered`?

File: src/geojs-scene-manager/index.js [hunk 3]
```diff
@@ -94,75 +89,52 @@ export class GeoJSSceneManager {
...
-      tooltip.position(evt.mouse.geo);
     });
-    lines.geoOn(geo.event.feature.mouseoff, evt => {
-      if (onNode) {
-        return;
-      }
 
-      tooltipElem.classList.toggle('hidden', true);
-    });
+    // NOTE: disable line hovering for now till figured out where to show
```
Comment (waxlamp) [inline src/geojs-scene-manager/index.js]: Can you file an issue to describe what needs to be done here? Or, if you've created the issue already, could you just link it here?

Comment (sgratzl) [reply src/geojs-scene-manager/index.js]: see https://github.com/arclamp/materialnet/issues/87

**Pull request merged**